packages/restlessness-cli/src/commands/deploy/index.ts

```diff
@@ -56,11 +63,11 @@ export default async (argv: minimist.ParsedArgs) => {
     const servicePath = path.relative(process.cwd(), path.join(PathResolver.getServicesJsonPath, `${serviceName}.json`));
 
     console.log(chalk.blue('Restlessness:'), 'Packaging service', serviceName);
-    const packageArgs = ['--config', servicePath, 'package', '--package', packagePath, '--stage', deploymentStage, '--verbose'];
+    const packageArgs = ['--config', servicePath, 'package', '--package', packagePath, '--stage', deploymentEnv, '--verbose'];
```

Comment:
the stage here is not equal to the env, it should be read from the JSON file stage var